emote/algorithms/sac.py

```diff
@@ -267,10 +267,10 @@ def __init__(
         lr_schedule: Optional[optim.lr_scheduler._LRScheduler] = None,
         n_actions: int,
         max_grad_norm: float = 10.0,
-        entropy_eps: float = 0.089,
         max_alpha: float = 0.2,
         name: str = "alpha",
         data_group: str = "default",
+        t_entropy: float | Schedule = None,
```

Comment:
`float | Schedule | None = None` if it's optional. :P

Reply:
Ahh types will be the death of me